resume_2

```diff
@@ -1,186 +0,0 @@
-      }
-	  h2 {
-		  font-style: italic;
-	  }
-  </style>
-  <h1 class="contact">Nicholas C. Long<br>3222 NW 54th ST Apt B24<br>Seattle, WA 98107<br>614-325-2406<br><a href="mailto:logan.nichols@gmail.com">logan.nichols@gmail.com</a></h1>
-    <h2 class="paid">Professional Experience</h2>
-        <ul class="jobs">
-			<li class="fud">Restaurants<br>
-				<ul class="cook">
-					<li>Crafting meals to order</li>
-					<li>Maintaining Equipment for both appearance and performance</li>
-					<li>Exploring flavor combinations like a rogue alchemist</li>
-				</ul>
-			</li>
-			<li class="airWaves">Radio<br>
-				<ul class="proder">
-					<li>Production Assistant for local call-in talk show on NPR affliate in Columbus Ohio</li>
-					<li>Freelance Producer for NPR affliate in Dayton Ohio</li>
-				</ul>
-			</li>
-			<li class="liluns">Teaching<br>
-				<ul class="teach">
-					<li>Assisting in creating an environment of learning for 2-4 year olds in a Montessori setting</li>
-				</ul>	
-			</li>
-			<li class="growem">Farming<br>
-				<ul class="vegs">
-					<li>Feeding chickens and collecting eggs</li>
-					<li>Cultivating a wide variety of vegetables over a three acre plot</li>
-					<li>Feeding and caring for pigs</li>
-					<li>Packaging produce for 45 local families weekly</li>
-				</ul>
-			</li>
-        </ul>
-	<h2 class="unPaid">Volunteer Experience</h2>
-	    <ul class="volt">
-			<li class="artWork">Pan-Eros<br>
-				<ul class="eros">
-					<li>Managing Security team for internationally renowned Art Festival</li>
-					<li>Cataloging Foundation Art</li>
-					<li>Training and tasking Volunteers</li>
-				</ul>
-			<li class="olymps">Special Olympics<br>
-				<ul class="coach">
-					<li>Coaching: Basketball Individual Skills Training, Swimming, and Softball</li>
-					<li>Participating as a Unified Partner: Swimming, Softball</li>
-					<li>Annoucing: Basketball</li>
-				</ul>
-			</li>
-	    </ul>
-	<h2 class="tellem">References</h2>
-	    <ul class="refs">
-	        <li class="Sophia">Sophia Iannicelli<br><p class="frankie">Director, <a href="http//:www.pan-eros.org">Pan-Eros Foundation</a>, <a href="mailto:director@seattleerotic.org">director@seattleerotic.org</a></p></li>
-	        <li class="Mike">Mike Thompson<br><p class="toms">News Director, <a href="https//:www.wosu.org">WOSU Public Media</a>, <a href="mailto:mike.thompson@wosu.org">mike.thompson@wosu.org</a></p></li>
-	        <li class="chefs">Andre Decker/Colleen Carrol<br><p class="chiefs">Andre is the Head Chef for Daniel's Broiler Downtown. <a href="mailto:adecker@schwartzbros.com">adecker@schwartzbros.com</a><br>Colleen is the Head Chef for Daniel's Broiler Leschi. <a href="mailto:ccarrol@schwartzbros.com">ccarrol@schwartzbros.com</a></p></li>
-	    </ul>
-	<h2 class="learnt">Education</h2>
-	    <ul class="knows">
-	        <li class="learn">Ohio State<br><p class="deg">In December of 2007, I graduated with a B.A. in English with a minor in theatre</p></li>
```

Comment:
I think you use `class` more for treating groups of items (like all the list items at a second or third level), `id` attribute is more for the individual pieces of text so you can reference a specific piece